### PR TITLE
feat(client): :sparkles: Add Pico 4 Ultra support

### PR DESCRIPTION
**Pull request by zmerp**

### CODE REVIEW

File: alvr/client_core/src/platform/mod.rs [hunk 1]
```diff
@@ -97,7 +103,10 @@ pub fn platform() -> Platform {
             ("Oculus", _, "seacliff") => Platform::QuestPro,
             ("Oculus", _, _) => Platform::QuestUnknown,
             ("Pico", "Pico Neo 3" | "Pico Neo3 Link", _) => Platform::PicoNeo3,
-            ("Pico", _, _) => Platform::Pico4,
+            ("Pico", _, "phoenix") => Platform::Pico4,
```
Comment (The-personified-devil) [inline alvr/client_core/src/platform/mod.rs:106]: Where did you get those codenames from?

**Review by The-personified-devil — APPROVED** [review]

File: alvr/client_core/src/platform/mod.rs (hunk 1 above)
Comment (zmerp) [reply alvr/client_core/src/platform/mod.rs:106]: A sketchy french driver website lol

Comment (zmerp) [reply alvr/client_core/src/platform/mod.rs:106]: I looked into the download filename https://www.touslesdrivers.com/index.php?v_page=12&v_code=2301

Comment (zmerp) [reply alvr/client_core/src/platform/mod.rs:106]: it's correct for Pico 4 Ultra so I think it's correct also for Pico 4 and G3

Comment (barnabwhy) [reply alvr/client_core/src/platform/mod.rs:106]: Pico 4 is in fact phoenix, I imagine the other two are correct too

**Pull request merged**